grakn-engine/src/main/java/ai/grakn/engine/controller/SystemController.java

```diff
@@ -69,8 +69,12 @@
  */
 public class SystemController {
     private final Logger LOG = LoggerFactory.getLogger(SystemController.class);
+    private final Properties graphProperties;
+    private final EngineGraknGraphFactory factory;
 
-    public SystemController(Service spark) {
+    public SystemController(Service spark, Properties graphProperties) {
+        this.graphProperties = graphProperties;
+        this.factory = EngineGraknGraphFactory.create(graphProperties);
```

Comment:
Can we create a method on the `EngineGraknGraphFactory` that will return the properties? That way we will no have to instantiate another one here.